packages/world/src/systems/NatureSystem.sol

```diff
@@ -75,15 +74,13 @@ contract NatureSystem is System {
 
     // This is enough to respawn the resource block, as it will be read from the original terrain next time
     EntityObjectType._deleteRecord(entityId);
-    EntityPosition._deleteRecord(entityId);
```

Comment:
Why dont we delete from `EntityPosition` here?

Reply:
because the entity is always in the same position for terrain